apps/web-app/package.json

```diff
@@ -16,10 +16,10 @@
         "@emotion/react": "^11.11.3",
         "@emotion/styled": "^11.11.0",
         "@semaphore-protocol/core": "4.3.1",
-        "@semaphore-protocol/data": "4.3.1",
```

Comment:
There is no need to update these versions.

Reply:
Could you please tell me how to remove package.json and yarn.lock , should I delete from my repo?

Reply:
Hey! 

You can delete the `package-lock.json` file from your repo. 

Regarding the `yarn.lock` file, if you keep the current versions of all the packages, everything should be fine and you won't need to remove this file. To keep the current versions of all the packages you need to have this: `"@semaphore-protocol/data": "4.3.1"` instead of this: `"@semaphore-protocol/data": "^4.4.1"` and `"next": "14.1.0"` instead of `"next": "^14.2.15"` in the `package.json` file.

Please let me know if you have any other questions.

Reply:
okay i removed both and updated package.json file

Reply:
Hey! Great! Thank you! Please, could you create an empty `yarn.lock` file in the root of the project and run the command `yarn` and push the changes? The `yarn.lock` file should still be in the repo but this PR shouldn't modify it.

After that, I think the PR is ready.

Reply:
i did it anyway and pushed same yarn.lock which is present in boilerplate repo

Reply:
Hey! The new `yarn.lock` file is almost correct. If you run `yarn`, it will be updated and you can push the change. The change will be just adding a line at the end of the file.

The `yarn.lock` file should not be part of these files after you push the change: https://github.com/semaphore-protocol/boilerplate/pull/76/files

Right now it's there: https://github.com/semaphore-protocol/boilerplate/pull/76/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2de